### PR TITLE
resolves #2042 update features and limitations to reflect current state of converter

### LINKED ISSUE #2042: Update features and limitations documentation for Asciidoctor PDF 2.0
In preparation for 2.0, we need to review and update the features and limitations documentation (ROOT:features.adoc). **A PR for this issue can't be submitted until https://github.com/asciidoctor/asciidoctor-pdf/pull/2035 has been merged.**

- Review both sections on ROOT:features.adoc and update to reflect Asciidoctor PDF 2.0.
- Add xrefs to related docs pages as needed

### PR DESCRIPTION
**Pull request by mojavelinux**

### CODE REVIEW

**Comment by mojavelinux** [comment]
@graphitefriction So far I've just updated the facts. I haven't added xrefs to relevant pages. If you'd like to do that, just add another commit with those additions.

File: docs/modules/ROOT/pages/features.adoc [hunk 1]
```diff
@@ -34,23 +34,24 @@
...
+* An inline image with a percentage `width` value in an `autowidth` table cell is resized relative to its intrinsic width.
+The space reserved for the image matches its intrinsic width.
+This matches the behavior of HTML.
 * Prawn does not support double-wide box drawing glyphs correctly, so box drawings aren't aligned properly in verbatim blocks (see https://github.com/prawnpdf/prawn/issues/1002[prawn#1002^]).
-* Orphan / widow support is limited; a page break can occur between a section title and its section content, a table caption and the caption, etc.; use a manual page break to avoid.
+* Orphan / widow support is limited; for example, a page break may occur between a section title the first block of content
+ ** to avoid an orphaned section title, you can extend the converter and override the arrange_section methodarrange_section methodarrange_section methodarrange_section method
```
Comment (graphitefriction) [inline docs/modules/ROOT/pages/features.adoc:50]: I feel like something went wrong here with the "arrange_section methodarrange_section methodarrange_section methodarrange_section method"?

File: docs/modules/ROOT/pages/features.adoc [hunk 2]
```diff
@@ -4,27 +4,27 @@
 == Highlights
 
 * Direct AsciiDoc to PDF conversion
-* xref:theme:index.adoc[Configuration-driven theme]
-* Custom fonts (TTF or OTF)
+* Composable, xref:theme:index.adoc[configuration-driven theming system] (resembling basic CSS)
+* Themable AsciiDoc blocks (e.g., example, admonition, sidebar, etc)
```
Comment (graphitefriction) [inline docs/modules/ROOT/pages/features.adoc:8]: The "etc" is missing a period

**Comment by graphitefriction** [comment]
>  So far I've just updated the facts. I haven't added xrefs to relevant pages. If you'd like to do that, just add another commit with those additions.

@mojavelinux I'll take care of the xrefs in a separate PR. If you just fix the two lines I commented on, then I'm good with this PR.

File: docs/modules/ROOT/pages/features.adoc (hunk 1 above)
Comment (mojavelinux) [reply docs/modules/ROOT/pages/features.adoc:50]: :scream:

**Comment by mojavelinux** [comment]
All fixed!

**Review by graphitefriction — APPROVED** [review]
Awesome!

**Pull request merged**